packages/pwa-kit-dev/src/ssr/server/build-dev-server.test.js

```diff
@@ -677,33 +676,82 @@ describe('DevServer rendering', () => {
 })
 
 describe('DevServer service worker', () => {
-    const TestFactory = {
-        ...NoWebpackDevServerFactory,
-        _getWebpackAsset: jest.fn()
+    let tmpDir
+
+    beforeEach(async () => {
+        tmpDir = await fse.mkdtemp(path.join(os.tmpdir(), 'pwa-kit-test-'))
+    })
+
+    afterEach(() => {
+        rimraf.sync(tmpDir)
```

Comment:
optional nit: We can achieve the same using `fse.removeSync(tmpDir)`. But I also see we use `rimraf` in other files, so it makes sense to leave it for consistency.
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/f70270fd6b9039d8ae70a15efd346b098dae04ff/packages/pwa-kit-runtime/src/ssr/server/express.test.js#L412